tests/tests_pytorch/loops/test_loops.py

```diff
@@ -387,9 +380,7 @@ def training_step(self, batch, batch_idx):
             },
         },
         "epoch_loop.val_loop.state_dict": ANY,
-        "epoch_loop.val_loop.dataloader_progress": ANY,
-        "epoch_loop.val_loop.epoch_loop.state_dict": ANY,
-        "epoch_loop.val_loop.epoch_loop.batch_progress": ANY,
```

Comment:
double check remapping of keys in checkpoint necessary or not for migration, as discussed offline